doc/getting_started.md

```diff
@@ -145,24 +150,12 @@ There may be times when you would like to build the library for a different oper
 - i686-pc-linux-gnu
 
 The build steps for cross compilation are very similar to the native build steps listed above. Specify the desired architecture from the list above under by using the `host` flag, and include any necessary configuration flags on the `./configure` command:
+
 ```
 make -C depends host=<target_architecture>
 ./autogen.sh
 ./configure
 make check
 ```
-It is important to run `make check` after cross compiling to make sure that everything is running properly for your architecture. For some guidance on which configuration flags to run, you can refer to our [CI test file](../.github/workflows/ci.yml).
-
-## Calling from Python or Go
-
-The other option for integrating the Libdogecoin Essential API is by using the set of wrappers built from the library itself. To begin using them right out of the box, you can download the Python wrappers through `pip` with the following command:
-```
-pip3 install libdogecoin
-```
-Or download the Golang wrappers through `go get`:
-```
-go get -u github.com/jaxlotl/go-libdogecoin
-go mod tidy
-```
 
-For more information on integrating these into your project or building the wrappers manually, please refer to [bindings.md](bindings.md).
+It is important to run `make check` after cross compiling to make sure that everything is running properly for your architecture. For some guidance on which configuration flags to run, you can refer to our [CI test file](../.github/workflows/ci.yml).
```

Comment:
Do we (can we) have a 'make check' equivalent for cmake?

Reply:
hmm not sure. i don't think so off the top, maybe just `cmake --build . && ./tests`?

Reply:
this would be slightly different build process though since cmake isn't really working with depends dir.